packages/browser/package.json

```diff
@@ -21,7 +21,8 @@
     "tapable": "^1.1.3",
     "url-join": "4.0.1",
     "url-parse": "1.4.7",
-    "webdriverio": "^4.14.0"
+    "webdriverio": "^4.14.0",
+    "@types/webdriverio": "^4.10.1"
```

Comment:
Can you remove the `@types/webdriverio` from `devDependencies`, and also move `@types/tapable` to `dependencies` too?

Reply:
Sure @adierkens